eng/targets/Wix.Common.targets

```diff
@@ -27,19 +27,24 @@
     <OutputName Condition="'$(OutputNamePrefix)' != '' AND '$(OutputNameSuffix)' != ''">$(OutputNamePrefix)$(_GeneratedPackageVersion)$(OutputNameSuffix)</OutputName>
 
     <EmbedCab Condition="'$(EmbedCab)' == ''">yes</EmbedCab>
-    <Cabinet Condition="'$(Cabinet)' == ''">$(OutputName.Replace('-', '_')).cab</Cabinet>
     <InstallDir>$(ProductName)</InstallDir>
 
     <DefineConstants Condition="'$(Configuration)' == 'Debug'">$(DefineConstants);Debug</DefineConstants>
     <DefineConstants>$(DefineConstants);EmbedCab=$(EmbedCab)</DefineConstants>
-    <DefineConstants>$(DefineConstants);Cabinet=$(Cabinet)</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(OutputType)' == 'package' AND '$(Cabinet)' == '' ">
+    <Cabinet>$(OutputName.Replace('-', '_')).cab</Cabinet>
+    <Cabinet Condition=" '$(EmbedCab)' != 'yes' ">$(OutputName.Replace('_win', '')).cab</Cabinet>
```

Comment:
This is just defence-in-depth. Nothing should depend in the CAB name whether it's embedded or not.

Reply:
Side note: This will affect the runtime pack as well. I don't see a problem there but want to call it out.

Reply:
should be fine until the cab is made external again, mostly because of the shared wixlib we produce to share the MSI between multiple installers